src/main/java/frc/robot/subsystems/swerveSubsystem/SwerveCommandFactory.java

```diff
@@ -197,10 +210,14 @@ public Command zeroDriveTrain() {
 
     return subsystem
         .runOnce(
-            () ->
-                subsystem.seedFieldRelative(
-                    new Pose2d(
-                        subsystem.getState().Pose.getTranslation(), Rotation2d.fromDegrees(180))))
+            () -> {
+              for (RotationSource RS : rotationSources) {
```

Comment:
Since `RS` isn't a constant, it should not be all upper case. I'd recommend calling the loop variable `rotationSource`.

Reply:
This loop could also be rewritten as a lambda:

```suggestion
              rotationSources.forEach(rs -> rs.zero());
```